.github/workflows/needsinfohelper.yaml

```diff
@@ -0,0 +1,13 @@
+name: Remove Labels
+
+on: [issue_comment]
+
+jobs:
+  remove_labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Do you need checkout? This will fire up entire git, or?

Reply:
For example this one does not use it:
https://github.com/nextcloud/android/blob/master/.github/workflows/autoApproveDependabot.yml

Reply:
thanks, that is me not yet familiar enough with the way github actions are working